pajbot/modules/wide_emote_limit.py

```diff
@@ -0,0 +1,121 @@
+import logging
+
+from pajbot.managers.handler import HandlerManager
+from pajbot.modules import BaseModule, ModuleSetting
+
+log = logging.getLogger(__name__)
+
+
+class WideEmoteLimitModule(BaseModule):
+    ID = __name__.split(".")[-1]
+    NAME = "Wide Emote Limit"
+    DESCRIPTION = "Times out users who post too many wide 7TV or FFZ emotes"
+    CATEGORY = "Moderation"
+    SETTINGS = [
+        ModuleSetting(
+            key="max_emotes",
+            label="Maximum number of wide emotes that can be posted",
+            type="number",
+            required=True,
+            placeholder="",
+            default=15,
+            constraints={"min_value": 1, "max_value": 167},
+        ),
+        ModuleSetting(
+            key="max_width",
+            label="Maximum width of emotes in pixels. Emotes exceeding this width will be counted as wide",
+            type="number",
+            required=True,
+            placeholder="",
+            default=128,
+            constraints={"min_value": 112, "max_value": 1000},
+        ),
+        ModuleSetting(
+            key="bypass_level",
+            label="Level to bypass module",
+            type="number",
+            required=True,
+            placeholder="",
+            default=420,
+            constraints={"min_value": 100, "max_value": 1000},
```

Comment:
Anything level 500+ is counted as mod, so `max_value` is pointless for anything over that. Probably best to set `max_value` to 499

Reply:
Adding @pajlada for opinion

Reply:
Sure

Reply:
This is false @ALazyMeme 
![image](https://user-images.githubusercontent.com/962989/183291893-7bab565f-53d4-49df-b944-69564caacf9c.png)

testaccount_420 or :joy: in this case is level 500 but not a moderator. moderator returns false, so keeping the max at 1000 is perfectly fine